demo_rails/app/views/application/googlecharts.erb

```diff
@@ -47,6 +47,17 @@ charts is not coming in 3d, since google_visualr loading only corechart package
 </p>
 <br>
 
+<h3> Importing Data from Google spreadsheet </h3>
+<p>
```

Comment:
It is better you show table and URL ,then display chart. I think we must show the simple code as well how it is working.

Reply:
I have added the URL and table. I have also added another example with easy data.